### PR TITLE
Added displaying coin cards list

### PR DESCRIPTION
**Pull request by rostyslav-onofruk**

### CODE REVIEW

File: src/components/Header/Header.tsx [hunk 1]
```diff
@@ -1,44 +1,74 @@
...
-                            Log out
-                        </Button>
-                    )}
-                    onLogoutSuccess={logout}
-                >
-                </GoogleLogout>
-            </Toolbar>
-        </AppBar>
+                    <GoogleLogout
+                        clientId="364643492164-lj63hjopspt4epse5ldmjq4t7fjgku6e.apps.googleusercontent.com"
```
Comment (vovabrenych) [inline src/components/Header/Header.tsx]: This clientID should store .env file and use thought Environment Variables
You can use this https://www.npmjs.com/package/dotenv or similar

File: src/interfaces/base.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
+export interface Base {
+    symbol: string;
+    sign: string;
+}
```
Comment (rostyslav-onofruk) [inline src/interfaces/base.ts]: test comment

File: src/pages/Home/Home.tsx [hunk 3]
```diff
@@ -1,88 +1,91 @@
...
-                {/*                {coin.description}*/}
-                {/*            </Typography>*/}
-                {/*        </CardContent>*/}
-                {/*        <CardActions>*/}
-                {/*            <div>*/}
-                {/*                More info <a href={coin.websiteUrl}>here</a>*/}
-                {/*            </div>*/}
-                {/*        </CardActions>*/}
-                {/*    </Card>*/}
-                {/*))}*/}
```
Comment (vovabrenych) [inline src/pages/Home/Home.tsx:85]: Please remove comments

File: src/components/Header/Header.tsx [hunk 4]
```diff
@@ -1,44 +1,76 @@
...
```
Comment (vovabrenych) [inline src/components/Header/Header.tsx]: This clientID should store .env file and use thought Environment Variables
You can use this https://www.npmjs.com/package/dotenv or similar

**Pull request merged**